### PR TITLE
pod priority class

### PR DESCRIPTION
**Pull request by lumi017**
Pod priority class support

### CODE REVIEW

File: kube-eagle/templates/deployment.yaml [hunk 1]
```diff
@@ -31,6 +31,9 @@ spec:
 {{ toYaml .Values.podSecurityContext | indent 8 }}
       {{- end }}
       serviceAccountName: {{ .Values.serviceAccountName }}
+{{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
```
Comment (weeco) [inline kube-eagle/templates/deployment.yaml:35]: I believe it must/should be quoted. What do you think about this?

```
{{ .Values.priorityClassName | quote }}
```

**Review by weeco — COMMENTED** [review]
Thanks for your PR, I added one comment. I am ready to package once that's resolved :-).

File: kube-eagle/templates/deployment.yaml (hunk 1 above)
Comment (lumi017) [reply kube-eagle/templates/deployment.yaml:35]: quotes are not required https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority

**Comment by weeco** [comment]
Merged, thanks for submitting the PR

**Pull request merged**